README.md

```diff
@@ -34,9 +36,49 @@ For alternative ways, stop by our Community Hub [stackrox.io](https://www.stackr
     + [How to Deploy](#how-to-deploy)
 * [Generating portable installers](#generating-portable-installers)
 
-## Quick Installation via Helm
+## Installation via Helm
+
+### Quick Installation
 
 StackRox offers quick installation via Helm Charts. Follow the [Helm Installation Guide](https://helm.sh/docs/intro/install/) to get `helm` CLI on your system.
+Then run the helm quick install script or proceed to section [Manual Helm Installation](#manual-helm-installation) for configuration options.
+```sh
+/bin/bash "$(curl --fail https://raw.githubusercontent.com/stackrox/stackrox/master/scripts/quick-helm-install.sh)"
```

Comment:
Several things here:
- I think you need `-c` to bash to make it work
- I suggest adding `-sSL` to curl params (short or long form, both is okay)

See yourself what happens when you call a resource that does not exist:
- `/bin/bash -c "$(curl --fail  https://raw.githubusercontent.com/stackrox/stackrox/non-existing/scripts/quick-helm-install.sh)"`
- `/bin/bash -c "$(curl --fail  -s https://raw.githubusercontent.com/stackrox/stackrox/non-existing/scripts/quick-helm-install.sh)"`
- `/bin/bash -c "$(curl --fail -sS https://raw.githubusercontent.com/stackrox/stackrox/non-existing/scripts/quick-helm-install.sh)"`
- (No example for `-L` but trust me, we want curl to follow redirections on codes 301 and 302)
- The same but no `-c` for bash: `/bin/bash -c "$(curl -fsSL  https://raw.githubusercontent.com/stackrox/stackrox/non-existing/scripts/quick-helm-install.sh)"`

Reply:
Just read through the man pages of bash and curl a bit, I'll take the -c option and -fsSL options.